client.go

```diff
@@ -110,6 +110,61 @@ func chopContentType(contentType string) string {
 	return strings.Split(contentType, ";")[0]
 }
 
+func (lc *StreamClient) Subscribe(ctx context.Context, group string, f func(ctx context.Context, payload []byte, contentType string) error) error {
```

Comment:
Should create a type for the callback fn

Reply:
done!